orchestra/migrations/0018_auto_20151007_1451.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
```

Comment:
high-level: i worry about how automatic the three migrations are.  if someone forgets to add their workflows to the filesystem, for example we might kill a bunch of data and it wont' be recoverable.  any way to avoid this, perhaps by having people explicitly call each step?